contrib/systemd/superset-webserver.service

```diff
@@ -0,0 +1,36 @@
+ #
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[Unit]
+Description =Superset Webserver Daemon
+After =network.target postgresql.service mysql.service redis.service rabbitmq-server.service
+Wants =postgresql.service mysql.service redis.service rabbitmq-server.service
```

Comment:
Using `After` and `Want` do not impose hard requisites, yet they assume everything is deployed on the same instance (a really bad idea)

Reply:
I tried to created systemd files like `apache airflow`. Okay, you're right, I will modify it as soon as possible.